persist-tool/add.c

```diff
@@ -193,6 +213,9 @@ add_main(int argc, char *argv[])
         }
     }
   g_free(line);
+
+  dump_all_entries(self);
```

Comment:
This dump is need to keep here, otherwise all the entries in the modified persist file will be saved with in_use bit set to 0 --> when you call `persist-tool dump` to this persist file it won't print items. This is the internal behavior of persist-state

Reply:
Ok I see. I was thinking that is intended to be used to print all entries to screen after the add. That would have made sense even if it weren't too useful.

Can you rename ref_all_entries to avoid such confusion, and edit this reason into the commit message?

Reply:
renamed

Reply:
Thanks!